pallets/schemas/src/lib.rs

```diff
@@ -477,7 +517,9 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Adds a given schema to storage. The schema in question must be of length
+		/// Adds a given schema to storage. (testnet)
```

Comment:
what's the meaning of (testnet) - is it only valid for testnet?  I would have expected a feature flag

Reply:
Following the existing pattern in the pallet, it exists but the call is filtered in the Runtime on Mainnet. Not sure why it's done that way vs a feature flag; that decision predates me.